tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -211,7 +254,15 @@ def test_sends_expected_data_to_seer(self, mock_get_similarity_data: MagicMock)
                 "platform": "python",
             },
         )
-        get_seer_similar_issues(new_event, new_event.get_grouping_variants())
+        self.project.update_option("sentry:similarity_backfill_completed", 1)
+        with patch(
+            "sentry.grouping.ingest.seer.event_content_is_seer_eligible",
+            return_value=True,
+        ):
+            should_call_seer_for_grouping(new_event, new_event.get_grouping_variants())
+        get_seer_similar_issues(new_event)
+
+        assert new_event.data.get("stacktrace_string") is None
```

Comment:
See above - we don't need to pop the data out. That said, if you do want to test that it doesn't end up in the eventual event, IMHO that should be a separate test just for that.